Error: EMFILE: too many open files, open '/home/lauren/aports/testing/code-oss/src/vscode-1.85.0/out-build/vs/base/browser/ui/list/listView.js.map'
--- ./build/lib/optimize.ts.orig
+++ ./build/lib/optimize.ts
@@ -121,7 +121,7 @@
 }
 
 function toConcatStream(src: string, bundledFileHeader: string, sources: bundle.IFile[], dest: string, fileContentMapper: (contents: string, path: string) => string): NodeJS.ReadWriteStream {
-	const useSourcemaps = /\.js$/.test(dest) && !/\.nls\.js$/.test(dest);
+	const useSourcemaps = false;
 
 	// If a bundle ends up including in any of the sources our copyright, then
 	// insert a fake source at the beginning of each bundle with our copyright
--- ./build/lib/optimize.js.orig
+++ ./build/lib/optimize.js
@@ -103,7 +103,7 @@
     return code.replace('"$BASE_URL"', 'baseUrl');
 }
 function toConcatStream(src, bundledFileHeader, sources, dest, fileContentMapper) {
-    const useSourcemaps = /\.js$/.test(dest) && !/\.nls\.js$/.test(dest);
+    const useSourcemaps = false;
     // If a bundle ends up including in any of the sources our copyright, then
     // insert a fake source at the beginning of each bundle with our copyright
     let containsOurCopyright = false;
